backbone.nativeajax.js

```diff
@@ -7,10 +7,14 @@
 //     https://github.com/akre54/Backbone.NativeAjax
 
 (function (factory) {
-  if (typeof define === 'function' && define.amd) { define(factory);
-  } else if (typeof exports === 'object') { module.exports = factory();
-  } else { Backbone.ajax = factory(); }
-}(function() {
+  if (typeof define === 'function' && define.amd) { 
+    define(factory.bind(this, this.XMLHttpRequest));
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('xmlhttprequest').XMLHttpRequest);
```

Comment:
This shouldn't be necessary now. If it's only for testing, let's do that in the tests file.